pkg/configui/configui.go

```diff
@@ -77,16 +77,15 @@ func pageDefaults[S PrimitiveWithSetBorder](name string, pages *tview.Pages, wid
 	Groovy       Groovy    `toml:"groovy:omitempty"`
 */
 
-func BuildMainMenu(cfg *config.Instance, pages *tview.Pages, app *tview.Application) *tview.List {
-	pages.RemovePage("main")
+func BuildMainMenu(cfg *config.Instance, pages *tview.Pages, app *tview.Application, exitFunc func()) *tview.List {
 	debugLogging := "DISABLED"
 	if cfg.DebugLogging() {
 		debugLogging = "ENABLED"
 	}
 	mainMenu := tview.NewList().
 		AddItem("Debug Logging", "Change the status of debug logging currently "+debugLogging, '1', func() {
 			cfg.SetDebugLogging(!cfg.DebugLogging())
-			BuildMainMenu(cfg, pages, app)
+			BuildMainMenu(cfg, pages, app, exitFunc)
```

Comment:
Added exitFunc in order to make closing the app optional when exiting the config